src/main/java/org/apache/sysml/hops/codegen/SpoofCompiler.java

```diff
@@ -564,11 +564,15 @@ private static void rConstructCPlans(Hop hop, CPlanMemoTable memo, HashMap<Long,
 		
 		//generate cplan for existing memo table entry
 		if( memo.containsTopLevel(hop.getHopID()) ) {
-			cplans.put(hop.getHopID(), TemplateUtils
-				.createTemplate(memo.getBest(hop.getHopID()).type)
-				.constructCplan(hop, memo, compileLiterals));
-			if( DMLScript.STATISTICS )
-				Statistics.incrementCodegenCPlanCompile(1); 
+			try {
+				cplans.put(hop.getHopID(), TemplateUtils
+						.createTemplate(memo.getBest(hop.getHopID()).type)
+						.constructCplan(hop, memo, compileLiterals));
+				if (DMLScript.STATISTICS)
+					Statistics.incrementCodegenCPlanCompile(1);
+			} catch (Exception e) {
+				LOG.error("problem constructing CPlan for hop "+hop.getHopID(), e);
+			}
```

Comment:
Apparently, SYSTEMML-1833 fixed this issue as well - I just gave it a try and could not reproduce any issue with stratstats anymore.